php/class-customize-snapshot-manager.php

```diff
@@ -1506,6 +1508,41 @@ public function render_templates() {
 				<p>{{ data.message }}</p>
 			</div>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-button">
+			<?php $title_text = __( 'click to expand', 'customize-snapshots' ); ?>
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
+			<span>
+				<?php esc_html_e( 'Snapshot conflicts', 'customize-snapshots' ); ?>
+				<a href="#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-{{id}}" class="dashicons dashicons-warning thickbox snapshot-conflicts-button" title="<?php echo esc_attr( $title_text ); ?>"></a>
+			</span>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-notification-template">
+			<ul>
+				<# _.each( data.notifications, function( notification ) { #>
+					<li class="notice notice-{{ notification.type || 'info' }} {{ data.altNotice ? 'notice-alt' : '' }}" data-code="{{ notification.code }}" data-type="{{ notification.type }}">{{{ notification.message || notification.code }}}</li>
+				<# } ); #>
+			</ul>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict">
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
```

Comment:
Why can't an HTML `id` attribute have brackets? There are no restrictions on what an `id` can have, except it cannot have spaces. You may be recalling HTML4 or XML which restricted the characters which can be in an ID. But browsers were always forgiving here, and HTML5 says the `id` attribute can contain anything. See https://www.w3.org/TR/html5/dom.html#the-id-attribute

![image](https://cloud.githubusercontent.com/assets/134745/18031791/811eeaea-6ca4-11e6-9496-c18350cee70a.png)